doc/devdocs/modules/keyboardmanager/keyboardeventhandlers.md

```diff
@@ -1,77 +1,106 @@
 # Keyboard Event Handlers (Remapping Logic)
+
 This file contains documentation for all the methods involved in key/shortcut remapping.
 
-## Table of Contents:
+## Table of Contents
+
+1. [GetSingleKeyRemapResult](#GetSingleKeyRemapResult)
```

Comment:
It's a valid Markdown to avoid reassigning numbers on inserting a new item in the future. If it looks strange, I can reassign increasing numbers. Or, given these headers are not necessarily ordered, how about using an unordered list?

Reply:
Unordered list is fine I believe

Reply:
Thank you for your confirmation. Updated with an unordered list.